### ISSUE TITLE
[1.19.2] [BUG] Render Error: Failed to build baked materially textured model for: Block{minecolonies:blockminecoloniesrack}

### ISSUE THREAD

**Issue by Sandriell** [ldtteam/minecolonies#9787]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Are you using the latest MineColonies Version?

- [X] I am running the latest **alpha** version of MineColonies for my Minecraft version.

### Did you check on the Wiki? or ask on Discord?

- [X] I checked the MineColonies Wiki and made sure my issue is not covered there. Or I was sent from discord to open an issue here.

### What were you playing at the time? Were you able to reproduce it in both settings?

- [ ] Single Player
- [X] Multi Player

### Minecraft Version

1.19.2

### MineColonies Version

minecolonies-1.19.2-1.1.467-BETA

### Structurize Version

structurize-1.19.2-1.0.689-RELEASE

### Related Mods and their Versions

Mod list: https://gist.github.com/Sandriell/765f5ba6b2853df86c6aba9be807576e

### Current Behavior

Console being spammed with: https://gist.github.com/Sandriell/7fa9095682c70352909d9f63a9dd5b96

Have also seen racks showing with a broken block texture:
![2024-02-18_21 27 48](https://github.com/ldtteam/minecolonies/assets/10704907/a4f5b1bf-c912-487d-85ee-e2a24ab1dd23)

### Expected Behavior

No spam, no broken textures

### Reproduction Steps

Unsure

### Logs

https://gist.github.com/Sandriell/7fa9095682c70352909d9f63a9dd5b96

### Anything else?

* Add a thumbs-up to the bug report if you are also affected. This helps the bug report become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by someaddons** [comment]
Its an issue of this mod:
`Caused by: java.lang.NullPointerException: Cannot invoke "net.minecraft.world.level.block.state.BlockState.m_61143_(net.minecraft.world.level.block.state.properties.Property)" because "state" is null
	at xyz.vsngamer.elevatorid.client.render.ElevatorBakedModel.getQuads(ElevatorBakedModel.java:68) ~[elevatorid-1.19.2-1.8.10.jar%23630!/:1.19.2-1.8.10] {re:classloading}
`

The state is here declared to be null able, the elevator or what mod it is isnt checking for null

**Comment by Sandriell** [comment]
Right, I meant to mention, but guess I forgot, that the bug report was more about having minecolonies handle the issue more gracefully- simply don't try and draw something that can't be rendered and not spamming the console.
